Khan/wonder-blocks
# Update Icon a11y docs to specify 3:1 contrast ratio

## Summary:
Turns out that icons can have a 3:1 contrast ratio, and
the article I initially used to say 4.5:1 misinterpreted
the spec. Thanks to Sarah B for pointing this out!

More info:
https://khanacademy.slack.com/archives/C8Z9DSKC7/p1700588780588889

Issue: non

## Test plan:
- Go to http://localhost:6061/?path=/docs/icon-accessibility--docs
- Confirm that the docs say 3:1 now
- Confirm that the new W3 reference navigates to the correct
  article.
